sktime/transformations/series/tests/test_kalman_filter.py

```diff
@@ -112,6 +113,14 @@ def rand_list(shape, length=ts, mult=10):
     ),  # [`ts` random ndarrays of shape (1,3)],
 }
 
+params_2_1_static = {
+    "state_transition": create_data((2, 2)),  # random ndarray of shape (2, 2)
+    "process_noise": create_data((2, 2)),  # random covariance matrix of shape (2, 2)
+    "measurement_function": create_data((1, 2)),  # random ndarray of shape (1, 2),
+    "measurement_noise": create_data((1, 1)),  # random covariance matrix of shape (1, 1)
+    "initial_state": create_data(2),  # random ndarray of shape (2,)
+    "initial_state_covariance": create_data((2, 2)),  # random covariance matrix of shape (2, 2)
```

Comment:
Should also add `covariance=True` if https://github.com/sktime/sktime/pull/8029 is merged